CMakeLists.txt

```diff
@@ -213,6 +216,8 @@ if (ENABLE_PARALLEL)
 
     if (METIS_MAJOR_VER)
       add_definitions(-DMETIS_${METIS_MAJOR_VER})
+    else (METIS_MAJOR_VER)
```

Comment:
Have a look at `FindMETIS.cmake` in losalamos/cinch@f4fb3233e8925a2a3618588d8ad8d787a17a28a8, it should be able to detect the metis version on the fly! Plus just append `Metis_DIR` to `CMAKE_PREFIX_PATH` and it will pick up that custom location and/or system library.